pkg/ingestor/puller/puller.go

```diff
@@ -68,7 +68,6 @@ func ExtractTarGz(archivePath string, basePath string, maxArchiveSize int64) err
 			return err
 		}
 		cleanPath := filepath.Join(basePath, header.Name) //nolint:gosec // We check the path just above
-		log.I.Infof("Extracting %s", cleanPath)
```

Comment:
Any idea why this was here before? If it was used for debugging, maybe replace it with a `log.I.Debugf` instead of removing it?

Reply:
I dont think it is useful now with the wrapper around it.